internal/proto/controller/api/resources/accounts/v1/account.proto

```diff
@@ -128,3 +133,39 @@ message OidcAccountAttributes {
   // Output only. userinfo_claims are the marshaled claims from userinfo.
   google.protobuf.Struct userinfo_claims = 130;
 }
+
+// Attributes associated only with Accounts with type "ldap".
+message LdapAccountAttributes {
+  // login_name of the authenticated user.  This is the login_name (or username)
+  // entered by the user when authenticating (typically the uid or cn
+  // attribute).  Account login names must be lower case.
+  string login_name = 100 [
```

Comment:
Nit: the other messages in this file start at a tag number of 10, is there a good reason for this to start at 100?

Reply:
see previous response.